### PR TITLE
Gidk 419 subheading issue

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
     public isBankUpdateInEdit: number = null;
     /** Update bank urn number */
     public updateBankUrnNumber: any;
-
+    public forceClearLinkAccount$: Observable<IForceClear> = observableOf({ status: false });
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:133]: Missing JS doc

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:133]: Who is responsible for this issue,,?, please fix this it today itself

File: apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html [hunk 2]
```diff
@@ -262,7 +266,7 @@ <h4 class="modal-title pull-left">Payment Status</h4>
                 <img src="{{imgPath}}payment-succes.svg">
             </div>
             <h4>Payment proceed</h4>
-            <p>Status of all payments will be mailed shortly.</p>
+            <p *ngIf="this.selectedAccForBulkPayment.length>1">Status of all payments will be mailed shortly.</p>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html:269]: remove "this"

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts (hunk 1 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:133]: @shalinee-walkover Please ask @ravinderwalkover ji to add JS DOC

**Pull request merged**